src/pick.c

```diff
@@ -291,10 +292,10 @@ selected_choice(void)
 		case DEL:
 			if (cursor_position > 0) {
 				delete_between(
-				    query,
-				    query_length,
-				    cursor_position - 1,
-				    cursor_position);
+					       query,
+					       query_length,
+					       cursor_position - 1,
+					       cursor_position);
```

Comment:
I think these instances were correctly indented. I use `cinoptions=:0,t0,+4,(4` in Vim. What editor and configuration do you use?